src/cluster.py

```diff
@@ -243,12 +254,36 @@ def member_started(self) -> bool:
         try:
             for attempt in Retrying(stop=stop_after_delay(60), wait=wait_fixed(3)):
                 with attempt:
-                    r = requests.get(f"{self._patroni_url}/health", verify=self.verify)
+                    r = requests.get(
+                        f"{self._patroni_url}/health",
+                        verify=self.verify,
+                        timeout=API_REQUEST_TIMEOUT,
+                    )
         except RetryError:
             return False
 
         return r.json()["state"] == "running"
 
+    @property
+    def member_replication_lag(self) -> str:
+        """Member replication lag."""
+        try:
+            for attempt in Retrying(stop=stop_after_delay(60), wait=wait_fixed(3)):
+                with attempt:
+                    cluster_status = requests.get(
+                        f"{self._patroni_url}/cluster",
```

Comment:
A lot of these URL magic strings are similar - can you extract them into variables please?

Reply:
Changed to a constant on [Extract endpoint from URL to constant](https://github.com/canonical/postgresql-operator/pull/32/commits/5adae46693502d3c53d20406138ba035c7836c81).